client/src/components/MultiJurisdictionAudit/StatusBox/index.tsx

```diff
@@ -78,16 +79,30 @@ const StatusBox: React.FC<IStatusBoxProps> = ({
   )
 }
 
-const downloadAuditAdminReport = (electionId: string) => {
-  apiDownload(`/election/${electionId}/report`)
-}
+const downloadAuditAdminReport = (electionId: string) =>
+  new Promise((resolve, reject) => {
+    apiDownload(`/election/${electionId}/report`)
+      .then(() => {
+        resolve('done')
+      })
+      .catch(err => {
+        reject(err)
+      })
+  })
```

Comment:
i dont think this change is necessary - since `apiDownload` returns a promise, then the code that was here previously is equivalent